### PR TITLE
feat(overlay): add global position strategy

### PR DESCRIPTION
**Pull request by jelbourn**
R: @hansl @robertmesserle @kara 

After much API discussion, this is what I went with. It's probable that this global strategy gets refactored once we actually start using it to build components and find it lacking.

### CODE REVIEW

File: src/core/overlay/position/global-position-strategy.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    this._cssPosition = 'fixed';
+    return this;
+  }
+
+  absolute() {
+    this._cssPosition = 'absolute';
+    return this;
+  }
+
+  top(value: string) {
```
Comment (hansl) [inline src/core/overlay/position/global-position-strategy.ts]: Should default value also be `0px`?

File: src/core/overlay/position/global-position-strategy.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    this._top = value;
+    return this;
+  }
+
+  left(value: string) {
+    this._left = value;
```
Comment (hansl) [inline src/core/overlay/position/global-position-strategy.ts]: You should set `this._right = ''` and `this._translateX = []` to counter `.center(...).left(...)` and `.right().left()`. Same with others.

**Comment by jelbourn** [comment]
@hansl addressed comments

**Comment by hansl** [comment]
LGTM.

**Comment by jelbourn** [comment]
@hansl removed `classList`

**Pull request merged**